issues/todo/ViMax_Complete_Integration_Package/src/pipelines/script2video_pipeline.py

```
@@ -0,0 +1,673 @@
+            chat_model=chat_model,
+            image_generator=image_generator,
+            video_generator=video_generator,
+            working_dir=config["working_dir"],
+        )
+
+    async def __call__(
+        self,
+        script: str,
+        user_requirement: str,
+        style: str,
+        characters: List[CharacterInScene] = None,
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)